applications/ChimeraApplication/tests/chimera_analysis_test.py

```diff
@@ -0,0 +1,55 @@
+import time
+import KratosMultiphysics.FluidDynamicsApplication as kfd
+import KratosMultiphysics.kratos_utilities as kratos_utilities
+have_external_solvers = kratos_utilities.CheckIfApplicationsAvailable("ExternalSolversApplication")
```

Comment:
are you sure you need this? => do you use SuperLU?
If so then better use a solver from the `EigenSolversApplication`, the `ExternalSolversApplication` will be removed in the long run

If you use amgcl you don't need this check at all, it is in the core

Reply:
I think this is a leftover from the time when using amgcl from external solvers app .. removed it.